parachain/pallets/control/src/lib.rs

```diff
@@ -76,14 +78,37 @@ pub mod pallet {
 		Upgrade { impl_address: H160, impl_code_hash: H256, params_hash: Option<H256> },
 		/// An CreateAgent message was sent to the Gateway
 		CreateAgent { location: Box<MultiLocation>, agent_id: AgentId },
+		/// An CreateChannel message was sent to the Gateway
+		CreateChannel { location: Box<MultiLocation>, para_id: ParaId, agent_id: AgentId },
```

Comment:
Not necessary to log the origin location. That is only required for CreateAgent

```suggestion
		CreateChannel { para_id: ParaId, agent_id: AgentId },
```